backend/src/test/java/de/bund/digitalservice/ris/norms/application/service/ElementServiceTest.java

```diff
@@ -116,4 +119,107 @@ void itReturnsEmptyOptionalIfNoElementIsFound() {
       assertThat(element).isEmpty();
     }
   }
+
+  @Nested
+  class loadElementHtmlFromNorm {
+    @Test
+    void itLoadsTheElementHtmlFromANorm() {
+      // Given
+      var eli = "eli/bund/bgbl-1/2000/s1/1970-01-01/1/deu/regelungstext-1";
+      var eid = "meta-1";
+
+      var normXml =
+          """
+              <?xml-model href="../../../Grammatiken/legalDocML.de.sch" schematypens="http://purl.oclc.org/dsdl/schematron"?>
+              <akn:akomaNtoso
+                xmlns:akn="http://Inhaltsdaten.LegalDocML.de/1.6/"
+                xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+                xsi:schemaLocation="http://Metadaten.LegalDocML.de/1.6/ ../../../Grammatiken/legalDocML.de-metadaten.xsd">
+                <akn:act name="regelungstext">
+                  <!-- Metadaten -->
+                  <akn:meta eId="meta-1" GUID="000">
+                    <akn:identification eId="meta-1_ident-1" GUID="000" source="attributsemantik-noch-undefiniert">
+                      <akn:FRBRExpression eId="meta-1_ident-1_frbrexpression-1" GUID="000">
+                        <akn:FRBRthis
+                          eId="meta-1_ident-1_frbrexpression-1_frbrthis-1"
+                          GUID="000"
+                          value="eli/bund/bgbl-1/2000/s1/1970-01-01/1/deu/regelungstext-1"
+                        />
+                      </akn:FRBRExpression>
+                    </akn:identification>
+                  </akn:meta>
+                </akn:act>
+              </akn:akomaNtoso>
+              """;
+
+      var norm = new Norm(XmlMapper.toDocument(normXml));
+      when(loadNormPort.loadNorm(new LoadNormPort.Command(eli))).thenReturn(Optional.of(norm));
+      when(xsltTransformationService.transformLegalDocMlToHtml(any())).thenReturn("<div></div>");
+
+      // When
+      var html =
+          service.loadElementHtmlFromNorm(new LoadElementHtmlFromNormUseCase.Query(eli, eid));
+
+      // Then
+      assertThat(html).contains("<div></div>");
```

Comment:
@hannes-mk Not sure if this is a useful test, but I also don't have a better idea right now ...

What I'm thinking:

- We want to test that the correct norm is loaded (ensured via the query in the `loadNorm` mock)
- We want to ensure it's converted to HTML
- We don't care about the specific HTML as this is covered by the tests of the converter, integration tests of the controller, and E2E tests

So based on that it would be fine but it still feels ... incorrect 🤨

Reply:
I stumbled across something similar in the Controller tests.

In this test we say: do return whatever comes back from the transformation service. Do not add anything and do not remove anything.

And this does not sound interesting or test worthy.
Let's keep it in mind for the general question of "what actually to test".